apps/server/src/modules/oauth/controller/oauth-sso.controller.ts

```diff
@@ -57,6 +59,34 @@ export class OauthSSOController {
 		res.redirect(errorRedirect.toString());
 	}
 
+	private migrationErrorHandler(
+		error: unknown,
+		session: ISession,
+		res: Response,
+		sourceSystemId: string,
+		targetSystemId: string
+	) {
+		this.logger.error(error);
```

Comment:
I am not sure with this logger.error(error) and the line 74 with this.logger.log(err);

It look like it is a special case with redirect but log is matched to info for now. Make not really sense from logger side, but is the current state. The logger.error will not be avaible in public interface of the logger. Because we need filter for each line that should be logged.

The complet flow is not in sync with our error pipline. In best case they should be used over inport to trigger the process directly. But this do not work for now. 

I expect that we must refactor it in future but maybe it is ok for now.

Reply:
ok, we can look at that/refactor in the future if changes happen.